include/alpaka/mem/view/ViewPlainPtr.hpp

```diff
@@ -313,10 +313,11 @@ namespace alpaka
                     dev::DevCudaRt>
                 {
                     //-----------------------------------------------------------------------------
+                    ALPAKA_NO_HOST_ACC_WARNING
```

Comment:
Shouldn't it be necessary on the method in line 288 as well?

Reply:
compiler did not complain, but from the code it should be there on 288 as you said.